app/components/chart_component/chart_component.html.erb

```diff
@@ -13,7 +13,16 @@
     <%= header %>
 
     <%= render 'shared/analysis_controls', chart_type: chart_type, axis_controls: axis_controls, analysis_controls: analysis_controls %>
-    <%= chart_tag(school, chart_type, no_zoom: no_zoom, wrap: false, html_class: html_class, chart_config: chart_config, fuel_type: fuel_type) %>
+    <% if dynamic_titles %>
```

Comment:
I don't think this option is needed as we've already got the `:header` slot on the component, see line 13.

So any templates that use a dynamic_titles section can set that to be the header slot and they will be added before the chart?

Reply:
this pr's started to get a bit unwieldy so I think leave in as part of this pr and remove as a separate bit of work?